spylib/request.py

```diff
@@ -111,6 +121,7 @@ def make_service_request(
         payload=None,
         timeout=2,
         retry_count=0,
+        retry_on_401_403=True,
```

Comment:
One thought here, we may eventually want to start using `kwargs` since this param list is growing. This could actually be a great candidate for a kwarg, since no user that consumes this function should ever really need to modify this setting (this will only need to be modified for internal usage I think). On that note though, I wouldn't prioritize back-tracking to fix this here, since our helper `post, put, get` etc all obfuscate the param anyways.

Reply:
:+1:  good idea